components/joinUsPage/buildYourCareer/BuildYourCareer.tsx

```diff
@@ -13,7 +13,7 @@ import useMediaQuery from "@mui/material/useMediaQuery";
 
 export const BuildYourCareer = () => {
   const data = useSelector((state: any) => state.joinUs.pageTexts);
-  const isMobile = useMediaQuery("(max-width:744px)");
+  const isShorterLine = useMediaQuery("(max-width:1440px)");
```

Comment:
```suggestion
  const isShorterLine = useMediaQuery(BREAKPOINTS.MD);
```

we need variable for responsive sizes maybe for overall project, but this can be done in context of other task